README.md

```diff
@@ -25,6 +25,33 @@ last-modified-at:
     date-format: '%d-%b-%y'
 ```
 
+You can also set `override-date: true` to have "last-modified-at" determine the datetime the file was first commited and use that to automatically set the post's `date` field.  Will use `ctime` if there's no git information.
```

Comment:
"override-date" is a bad name for this setting, since it actually doesn't override dates on e.g. posts that have been set via the file. Open to suggestions.

Reply:
Suggestions:
- `set-date`
- `set-page-date`